packages/@react-spectrum/overlays/src/Modal.tsx

```diff
@@ -58,10 +58,9 @@ let typeMap = {
 
 let ModalWrapper = forwardRef(function (props: ModalWrapperProps, ref: RefObject<HTMLDivElement>) {
   // eslint-disable-next-line @typescript-eslint/no-unused-vars
-  let {children, isOpen, type, isDismissable, isKeyboardDismissDisabled, ...otherProps} = props;
+  let {children, isOpen, type, overlayProps, ...otherProps} = props;
```

Comment:
This looks like an API change, but I think it is low enough that it doesn't count.

Reply:
it's internal :)